modules/trimmomatic/trimmomatic.nf

```diff
@@ -50,6 +50,17 @@ process TRIMMOMATIC {
 
 
     """
+    echo "trimmomatic \                                                                
+    PE \                                                                         
+    -threads ${task.cpus} \                                                      
+    -phred33 \                                                                   
+    ${genomeReads[0]} \                                                          
+    ${genomeReads[1]} \                                                          
+    $fq_1_paired \                                                               
+    $fq_1_unpaired \                                                             
+    $fq_2_paired \                                                               
+    $fq_2_unpaired \                                                        
+    LEADING:3 TRAILING:3 SLIDINGWINDOW:4:20 MINLEN:36" 
```

Comment:
Could you please run the workflow using `-stub -with-tower` and share that with me? I'd like to see the exact output of this command.

Reply:
replied on another PR, but you can check  `cranky_roentgen`  tower execution that I shared with you.

Reply:
Cool, I've checked and it works as expected 👍